src/frontend/js/modules/writing.js

```diff
@@ -52,11 +52,16 @@ export default class Writing {
       this.editor = editor;
     });
 
+    window.onbeforeunload = (e) => {
```

Comment:
What does it stand for?

Reply:
This is for exiting edit page confirmation. Browser does not allow to custom alert text so I just let it like that to work with all browsers.